### ISSUE TITLE
Running locally

### ISSUE THREAD

**Issue by alexyao2015** [1nikolas/play-integrity-checker-app#30]
It seems that for non playstore apps to request an integrity token, the `setCloudProjectNumber` field must be set [here](https://github.com/1nikolas/play-integrity-checker-app/blob/96c949d5fbf5d002cf529c1919b5458afc97dd68/app/src/main/java/gr/nikolasspyr/integritycheck/MainActivity.java#L75). Without this number set, it is not possible to use a custom api server. The cloud project number can be found [here](https://cloud.google.com/resource-manager/reference/rest/v1/projects/list) using the api tester. I'm curious if you would also consider adding this as a variable that can be accessed through the `local.properties` file, similar to the `API_URL` property.

Also the `API_URL` specified in the [readme](https://github.com/1nikolas/play-integrity-checker-app/blob/96c949d5fbf5d002cf529c1919b5458afc97dd68/README.md?plain=1#L19) is incorrect with a trailing `/` because the app itself already appends the `/`. Please consider correcting the documentation.

Lastly, in my use case, I modified the server a little bit so that I could use it outside of Vercel in a local Docker environment. With this, I ran into an issue where accessing my local server directly by IP in the local environment resulted in an error, so I had to add `android:usesCleartextTraffic="true"` to the manifest as specified [here](https://stackoverflow.com/questions/45940861). I understand if you may not want to add this to the main app, but it would be a helpful hint to also have in the documentation.

Thanks for making this wonderful app!

**Comment by 1nikolas** [comment]
> It seems that for non playstore apps to request an integrity token, the setCloudProjectNumber field must be set here.

Yes but non playstore apps can't check for strong and basic integrity, only device. So half of the UI would become useless.


> Also the API_URL specified in the readme is incorrect with a trailing / because the app itself already appends the /.

Done


> so I had to add android:usesCleartextTraffic="true"

This is only required if your server is running with http (not https) which I won't embrace.

**Comment by alexyao2015** [comment]
> Yes but non playstore apps can't check for strong and basic integrity, only device. So half of the UI would become useless.

I don't believe this is the case actually. I was able to see basic and device integrity pass with my local server. 

> Technically you don't even need a server but it's more secure that way

I guess for just a checker app, does the security really matter since there isn't any sort of "secure" method that you are checking?

Thanks for the quick update!

**Comment by 1nikolas** [comment]
> I don't believe this is the case actually. I was able to see basic and device integrity pass with my local server.
When I first made this app, this was how it worked. Now I have no idea

> I guess for just a checker app, does the security really matter since there isn't any sort of "secure" method that you are checking?
Works now, I'm not touching it :P
